### PR TITLE
(feat): add drain-node chart

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  add drain node chart

**Logs**
[logs.txt](https://github.com/litmuschaos/chaos-charts/files/3968170/logs.txt)

### CODE REVIEW

File: charts/generic/drain-node/node-drain.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    name: Mayadata
+  links:
+    - name: Kubernetes Website
+      url: https://kubernetes.io
+    - name: Source Code
+      url: https://github.com/litmuschaos/litmus/tree/master/experiments/generic
+  icon:
+    - url: ""
+      mediatype: ""
+  chaosexpcrdlink: https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/drain-node/experiment.yaml
```
Comment (uditgaurav) [inline charts/generic/drain-node/node-drain.chartserviceversion.yaml:37]: Please check this link is not working.

File: charts/generic/drain-node/node-drain.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    vendor: Mayadata
+    repository: https://github.com/litmuschaos/chaos-charts
+    support: https://app.slack.com/client/T09NY5SBT/CNXNB0ZTN
+spec:
+  displayName: node-drain
+  categoryDescription: >
+    Drain the node where application pod is scheduled
+  keywords:
+    - Kubernetes
+    - Node
```
Comment (uditgaurav) [inline charts/generic/drain-node/node-drain.chartserviceversion.yaml:17]: - State

File: charts/generic/drain-node/experiment.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        - "*"
+    image: "litmuschaos/ansible-runner:ci"
+    args:
+    - -c
+    - ansible-playbook ./experiments/generic/drain_node/drain_node_ansible_logic.yml -i /etc/ansible/hosts -vv; exit 0
+    command:
+    - /bin/bash
+    env:
+    - name: ANSIBLE_STDOUT_CALLBACK
+      value: default
```
Comment (uditgaurav) [inline charts/generic/drain-node/experiment.yaml]: 'default'

File: charts/generic/drain-node/node-drain.chartserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: Mayadata
+    repository: https://github.com/litmuschaos/chaos-charts
+    support: https://app.slack.com/client/T09NY5SBT/CNXNB0ZTN
+spec:
+  displayName: node-drain
+  categoryDescription: >
+    Drain the node where application pod is scheduled
```
Comment (uditgaurav) [inline charts/generic/drain-node/node-drain.chartserviceversion.yaml:14]: Check if we can add more details.

File: charts/generic/drain-node/node-drain.chartserviceversion.yaml (hunk 1 above)
Comment (ispeakc0de) [reply charts/generic/drain-node/node-drain.chartserviceversion.yaml:37]: This link will active after merging of this PR

**Review by ksatchit — APPROVED** [review]

**Pull request merged**